<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export::ExportButtonGroup</class>
 <widget class="QGroupBox" name="Export::ExportButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Export (all tracks)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="exportAsWavBtn">
     <property name="text">
      <string>Export as WAV files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportAsFlacBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Export as FLAC files</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="exportAsMp3Btn">
     <property name="text">
      <string>Export as MP3 files</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
